src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.Sdk.VisualBasic.Analyzers.targets

```diff
@@ -0,0 +1,25 @@
+<!--
+***********************************************************************************************
+Microsoft.NET.Sdk.VisualBasic.Analyzers.targets
+
+WARNING:  DO NOT MODIFY this file unless you are knowledgeable about MSBuild and have
+          created a backup copy.  Incorrect changes to this file will make it
+          impossible to load or build your projects from the command-line or the IDE.
+
+Copyright (c) .NET Foundation. All rights reserved. 
+***********************************************************************************************
+-->
+<Project ToolsVersion="14.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <PropertyGroup Condition="'$(DisableNetAnalyzers)' != 'true' And '$(_TargetFrameworkVersionWithoutV)' >= '5.0'">
```

Comment:
Use MSBuild version comparison intrinsic functions for version comparisons (line 17 as well)